packages/lexical-playground/src/nodes/ExcalidrawNode/index.tsx

```diff
@@ -29,13 +29,13 @@ const ExcalidrawComponent = React.lazy(() => import('./ExcalidrawComponent'));
 export type SerializedExcalidrawNode = Spread<
   {
     data: string;
-    width: Dimension;
-    height: Dimension;
+    width?: Dimension;
+    height?: Dimension;
   },
   SerializedLexicalNode
 >;
 
-function $convertExcalidrawElement(
+function convertExcalidrawElement(
```

Comment:
This shouldn't be renamed
```suggestion
function $convertExcalidrawElement(
```